config.py

```diff
@@ -0,0 +1,21 @@
+import yaml
+
+try:
+    config_file = open('./config/company_config.local.yml')
+except:
+    config_file = open('./config/company_config.yml')
```

Comment:
If you don't have REALLY working default configuration, you shouldn't have this. Fallback to non-working configuration makes UX worse. It's better to just show an error with instruction to the users.